chain/gap/notify.go

```diff
@@ -18,12 +18,12 @@ type Notifier struct {
 	queue                *queue.AsynQ
```

Comment:
changes here due to unify range flags (--to and --from) to use int64's. mechanical change